### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by psiinon**
Updating as we go through it 😉

### CODE REVIEW

**Comment by thc202** [comment]
In the "Pre release" phase (after line 15?), add e.g.: "Publish a `SNAPSHOT` of core and update the main add-ons to use it."

File: RELEASING.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ These tasks use checkboxes so that they can be copied into an issue.
 
 - [ ] Run the workflow [Prepare Release Main Version](https://github.com/zaproxy/zaproxy/actions/workflows/prepare-release-main-version.yml),     to prepare the release. It creates a pull request updating the version;
 - [ ] Finish the following tasks in the pull request:
+  - [ ] Update latest ZAP version in [build.gradle.kts](https://github.com/zaproxy/zap-admin/blob/master/build.gradle.kts)
```
Comment (thc202) [inline RELEASING.md:26]: `ZAP version`  → `ZapVersions file`

**Comment by psiinon** [comment]
Done those :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**